src/components/Article.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Article(info) {
```

Comment:
Делай деструктуризацию props
```suggestion
function Article({ title, date, text }) {
```
Передавай три props в компонент Article